README.md

```diff
@@ -177,3 +177,11 @@ As a participant, you will need an account on each of these services.
 
 [Open an issue](https://github.com/ngi-nix/summer-of-nix/issues) to ask a question in public.
 Send us direct messages on Matrix for questions that require privacy.
+
+## Funding
+
+Summer of Nix is supported by [NLnet](https://nlnet.nl/foundation/), with financial backing from the European Commission's [Next Generation Internet](https://ngi.eu/about/) program.
+
+[<img src="https://nlnet.nl/logo/banner.svg" alt="NLnet foundation logo" width="30%" />](https://nlnet.nl/foundation/)
+<span style="margin: 10px"></span>
+[<img src="https://nlnet.nl/image/logos/EC.svg" alt="European Commission logo" width="20%" />](https://ngi.eu/about/)
```

Comment:
```suggestion
[<img src="https://nlnet.nl/image/logos/EC.svg" alt="European Commission logo" />](https://ngi.eu/about/)
[<img src="https://nlnet.nl/logo/banner.svg" alt="NLnet foundation logo" />](https://nlnet.nl/foundation/)
```

None of this registers in GitHub markdown, and we're not rendering it anywhere else. Also let's put the EC first.